Source/Parallelization/GuardCellManager.H

```diff
@@ -0,0 +1,42 @@
+#ifndef GUARDCELLMANAGER_H_
+#define GUARDCELLMANAGER_H_
+
+#include <AMReX_IntVect.H>
+
+class guardCellManager{
+
+public:
+
+    int Init(
+        const bool do_subcycling,
+        const bool do_fdtd_nci_corr,
+        const bool do_nodal,
+        const bool do_moving_window,
+        const bool do_fft_mpi_dec,
+        const bool aux_is_nodal,
+        const int moving_window_dir,
+        const int nox,
+        const int nox_fft, const int noy_fft, const int noz_fft,
+        const int nci_corr_stencil,
+        const int maxwell_fdtd_solver_id,
+        const int max_level);
+
+    // Guard cells allocated for each multifab
+    amrex::IntVect ng_alloc_EB = amrex::IntVect::TheZeroVector();
+    amrex::IntVect ng_alloc_J = amrex::IntVect::TheZeroVector();
+    amrex::IntVect ng_alloc_Rho = amrex::IntVect::TheZeroVector();
+    amrex::IntVect ng_alloc_F = amrex::IntVect::TheZeroVector();
+    int ng_alloc_F_int = 0;
```

Comment:
Is it required to keep track of `ng_alloc_F_int` on top of `ng_alloc_F`?

Reply:
I got rid of `ng_alloc_F_int`